### ISSUE TITLE
More Documentation Needed

### ISSUE THREAD

**Issue by kevinwu** [inspirit/jsfeat#58]
The documentation here (https://inspirit.github.io/jsfeat/#structs) is not very helpful for people new to jsfeat. Some getting started guides would be helpful.

For example, I have a webcam stream using getUserMedia. I want to apply a Gaussian Blur on that livestream. There is a jsfeat demo doing just that but I have no idea how to implement it in my code as there is no documentation on how to get started.

Can anyone help?

**Comment by Nateowami** [comment]
I agree, the documentation could be more helpful. However, if you view source on the examples, you can get a good idea of how to use it. Just search the source for `jsfeat` and see what is being called, and what arguments are being passed in. It's not perfect, but it got me unstuck. Kinda.

**Comment by schnell9** [comment]
Take a look:
https://www.packtpub.com/web-development/computer-vision-web

**Comment by alexreflexive** [comment]
Your learn a lot with Computer Vision. Good explanation about how the all thing works. But to write real code it's still quite difficult.
